WebHostLib/tracker.py

```diff
@@ -260,8 +260,18 @@ def get_static_room_data(room: Room):
     names: Dict[int, Dict[int, str]] = multidata["names"]
     groups = {}
     if "slot_info" in multidata:
+        games = {slot: slot_info.game for slot, slot_info in multidata["slot_info"].items()}
         groups = {slot: slot_info.group_members for slot, slot_info in multidata["slot_info"].items()
                   if slot_info.type == SlotType.group}
+
+        for game in games.values():
+            if game in multidata["datapackage"]:
+                lookup_any_location_id_to_name.update(
```

Comment:
these are globals and shared between multiple Rooms, which can be problematic.

Reply:
I was originally using a completely separate dictionary and updating that with the new names, then using `any.get(id, custom.get(id, id))`. should i just create a copy of the global and update that instead? seems better.

Reply:
I'd recommend keeping the global lookup and then also extracting the special data as static room data, then using chain() for lookups.

Reply:
looks like the custom trackers don't use the filter at all, so i build the filter right before rendering of the generic tracker. couldn't figure out how to pass it as an arg from searching. did check that the multi and all the custom trackers still work with this change though 🤷